wg-administrative/initiatives/OpenJS.md

```diff
@@ -35,4 +35,3 @@ We will strive for unanimous consensus, and failing that will look for majority
 - [@groundwater](https://github.com/groundwater)
 - [@jkleinsc](https://github.com/jkleinsc) (initial chair)
 - [@lee-dohm](https://github.com/lee-dohm)
-- [@nornagon](https://github.com/nornagon)
```

Comment:
I think this is fine to keep as a historical document. Many of the people listed here are emeriti already.